src/components/Editor/Editor.css

```diff
@@ -92,6 +92,10 @@ html[dir="rtl"] .editor-mount {
   );
 }
 
+.new-breakpoint .CodeMirror-linenumber {
+  pointer-events: none;
+}
```

Comment:
this was essential for the Editor/Breakpoint.js onClick and onContextMenu to work!